internal/btf/types.go

```diff
@@ -159,6 +216,7 @@ func (e *Enum) copy() Type {
 type Fwd struct {
 	TypeID
 	Name
+	KindFlag bool
```

Comment:
I forgot to address this: from https://github.com/torvalds/linux/commit/9d5f9f701b1891466fb3dbb1806ad97716f95cc3#diff-fa650a64fdd3968396883d2fe8215ff3 it seems that this determines whether the forward declaration is for a struct or union. I think a dedicated enum like IntEncoding would be nicer here than the boolean. If you don't need this for your patches feel free to remove KindFlag and punt on this as well.

Reply:
I haven't needed it thus far, so I'll remove it for now.